Fixing retrieval of TransientTrackBuilder

dzuolo opened the pull request:
I fixe the retrieval of TransientTrackBuilder according to the last CMSSW implementation of these kind of object.
I also removed the GlobalTrackingGeometryRecord since it was not used in the code. 
FYI @francescobrivio

dzuolo committed:
Fixing retrieval of TransientTrackBuilder according to the last CMSSW implementation of these kind of object

francescobrivio commented on errorScaleCal/plugins/errorScaleCal.cc:
```diff
@@ -125,11 +127,7 @@ errorScaleCal::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
 
-
-  edm::ESHandle<TransientTrackBuilder>            theB                ;
-  edm::ESHandle<GlobalTrackingGeometry>           theTrackingGeometry ;
-  iSetup.get<GlobalTrackingGeometryRecord>().get(theTrackingGeometry) ;
```
@dzuolo could you also remove the include for the GlobalTrackingGeometryRecord?
`#include "Geometry/Records/interface/GlobalTrackingGeometryRecord.h"`

dzuolo replied:
Removed in https://github.com/MilanoBicocca-pix/BeamSpotCalibration/pull/4/commits/dc72fb88814b7907a221b48aa4b2be4319749e2a

dzuolo committed:
Minor fix after removing some unused objects 

Removed ```#include "Geometry/Records/interface/GlobalTrackingGeometryRecord.h"```
